drud/ddev
# Convert from packr2 to go:embed, fixes #2343

## The Problem/Issue/Bug:

#2343: go v1.16 gives us go:embed, so we can get rid of packr2 and the problems we had with it.

`make packr2` is no longer required

And packr2 usage anywhere but amd64 mac was giving trouble. It produced varying results when executed on Windows or even on arm64 macOS

## Manual Testing Instructions:

`ddev start` should:
* Create settings files for all project types
* Create apache/nginx configs for all project types
* Create default commands for all project types, including in global
* Verify that the nginx config for each project type is correct

## Automated Testing Overview:

- [x] Tests are there for all the commands
- [x] Add tests to make sure the apache/nginx stuff gets set up?
- [x] settings for drupal/magento/etc are covered by TestDdevFullSiteSetup, since they can't work without the settings.

## Related Issue Link(s):

#2343
